server/src/main/java/com/cloud/ha/HighAvailabilityManagerImpl.java

```diff
@@ -608,7 +608,9 @@ protected Long restart(final HaWorkVO work) {
 
             VMInstanceVO started = _instanceDao.findById(vm.getId());
             if (started != null && started.getState() == VirtualMachine.State.Running) {
+                String message = String.format("HA on VM: %s", started.getHostName());
                 s_logger.info("VM is now restarted: " + vmId + " on " + started.getHostId());
```

Comment:
Taking the opportunity, what do you think of enhancing also the INFO log message?

As a suggestion, it could be logged something as the following:
```
HA is now restarting VM instance {id: "123", name: "VM Name", uuid: "1234.....abc", type="User"} on Host [{id: "1", name: "host.name", uuid: "ABC...123", type="Routing"}]
```
With the following code changes:

```suggestion
                HostVO hostVmHasStarted = _hostDao.findById(started.getHostId());
                s_logger.info(String.format("HA is now restarting %s on %s", started, hostVmHasStarted));
```